amr-wind/wind_energy/actuator/FLLC.cpp

```diff
@@ -38,6 +42,70 @@ void FLLCInit(
     data.dr[0] = vs::mag(view.pos[1] - view.pos[0]) / 2.;
     data.dr[npts - 1] = vs::mag(view.pos[npts - 1] - view.pos[npts - 2]) / 2.;
 
+//amrex::OutStream() << "Error NOT Here 1"  << std::endl;
+    namespace interp = ::amr_wind::interp;
+
+    // Non-uniform radial distribution variables
+    size_t idx = 0;
+    amrex::Real dr;
+    amrex::Real dr1;
+    amrex::Real eps1;
+    amrex::Real dr2;
+    amrex::Real eps2;
+
+//amrex::OutStream() << "Error NOT Here 2"  << std::endl;
+
+//amrex::OutStream() << data.nonuniform_r  << std::endl;
+//amrex::OutStream() << data.dr[0]  << std::endl;
+//amrex::OutStream() << view.vel_pos[0]  << std::endl;
+//amrex::OutStream() << data.span_distance_vel[0]  << std::endl;
+
+    // Non-uniform radial distribution calc
+    data.nonuniform_r.push_back(
+        data.r[0]
+); // Initialize the first location
+//amrex::OutStream() << "Error NOT Here nonu back " << data.nonuniform_r.back()  << std::endl;
+//amrex::OutStream() << "Error NOT Here back " << data.r.back()  << std::endl;
+    while (data.nonuniform_r.back() < data.r.back()) {
+        amrex::Real r_ = data.nonuniform_r.back(); // The latest radial position
+        // Interpolate the value of epsilon to the current radial location
+        eps1 = interp::linear(data.r, data.optimal_epsilon, r_);
```

Comment:
This should be interpolating the value at the last radial location `r_` in every iteration. 
This needs to be done inside the loop because the value keeps changing. 
Every iteration in this loop will append a new radial location. That is how the nonuniform grid is constructed. 
The linear interpolation function is overloaded and in this case it should return only one value.